part-5/qualification.md

```
@@ -94,13 +96,13 @@ Describes all projects used in a qualification scenario. Currently, only PK-Sim
 
   - "**TargetSimulations**": Simulation name(s) within child project
 
-#### Example
+##### Example
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)